public/app/features/panel/panel_header.ts

```diff
@@ -26,6 +26,12 @@ var template = `
     </ul>
   </span>
   <span class="panel-time-info" ng-show="ctrl.timeInfo"><i class="fa fa-clock-o"></i> {{ctrl.timeInfo}}</span>
+  <span class="panel-time-info" ng-show="ctrl.panel.soloMode">
+    <i class="fa fa-clock-o"></i>
+    {{ctrl.timeRangeString}}
+    <span class="gf-timepicker-utc" ng-show="ctrl.dashboard.isTimezoneUtc()">UTC</span>
+    <span class="gf-timepicker-utc" ng-show="!ctrl.dashboard.isTimezoneUtc()">{{ctrl.dashboard.getTimezone()}}</span>
+  </span>
```

Comment:
can you remove this change? It seems unrelated to the PR

Reply:
the previous codebase had a query-parameter `tz` which was mostly set to `UTC` that parameter was sent to phantom using environment variables. Since the chromium process is reused between requests we cannot specify the timezone via environment variable which also means that the `tz` request parameter is removed (until it is supported in `d-solo`).

As the timezone can no longer be specified the rendered image will have the timezone as specified in the dashboard preference and *not* as specified via the request parameter (no longer supported). In order to solve that I opted for just putting the timing information and timezone information into the rendered image.